lib/Yam/Agama/Pom/RebootPage.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Handles installation reboot screen.
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+package Yam::Agama::Pom::RebootPage;
+use strict;
+use warnings;
+
+use testapi;
+
+sub new {
+    my ($class, $args) = @_;
+    return bless {
+        tag_installation_complete => 'agama-install-finished',
+        tag_reboot_button => 'reboot'
+    }, $class;
+}
+
+sub expect_is_shown {
+    my ($self, $timeout) = @_;
```

Comment:
please, use here `$args` and then get $args->timeout, so from outside the caller can put something like `{ timeout => X }` in our libyui-rest-api code we use this a lot, although in the repo you would not see it that frequent, but those position parameter a a bit magical, better named one, special if the name of the function doesn't really connect it, like "shown with time" for example.